project-koku/koku
# Cost 2554 trino processing pg pk violation

Fixes [COST-2554](https://issues.redhat.com/browse/COST-2554)

Changes proposed in this PR:
* Generate uuids when updating PostgreSQL instead of temp storage time
* Update unit tests

Testing Instructions:
1. Checkout branch and process data via trino. There should be a net-zero effect on the data generated

Smoke Test Results:
```
=========================== short test summary info ============================
  FAILED tests/rest_api/v1/test_source.py::test_api_ocp_on_aws_source_crud - as...
  FAILED tests/rest_api/v1/test_source.py::test_api_ocp_on_azure_source_crud - ...
= 2 failed, 621 passed, 2 skipped, 8385 deselected, 2 warnings in 1367.99s (0:22:47) =
```
[console output](https://ci.ext.devshift.net/job/project-koku-koku-pr-check/4838/console)
